src/annotator/guest.js

```diff
@@ -116,10 +116,10 @@ export default class Guest {
   constructor(element, eventBus, config = {}) {
     this.element = element;
     this._emitter = eventBus.createEmitter();
-    this.visibleHighlights = false;
+    this._visibleHighlights = false;
```

Comment:
That's a good question. I think they could be useful in scenarios where it is important to prevent access to private fields, such as if we are exposing an object to a third-party. Its less clear whether there is any benefit for most of our code where it just needs to be clear to other developers whether a field _should_ be considered private or public.